gtsfm/two_view_estimator.py

```diff
@@ -25,6 +30,13 @@
 pil_logger = logging.getLogger("PIL")
 pil_logger.setLevel(logging.INFO)
 
+BA_NOISE_MODEL_EPIPOLAR_ALGEBRAIC_ERROR = gtsam.noiseModel.Isotropic.Sigma(1, 0.01)
+
+DATA_ASSOC_2VIEW = DataAssociation(reproj_error_thresh=3, min_track_len=2, mode=TriangulationParam.NO_RANSAC)
+BUNDLE_ADJUST_2VIEW = BundleAdjustmentOptimizer(
+    output_reproj_error_thresh=100
+)  # we dont care about output error threshold
```

Comment:
If the reprojection error after BA is high, it's likely that the correspondences between the views are contaminated with outliers. Maybe this view pair should be rejected, or it at least needs further filtering?